src/components/RGBColorPicker/RGBColorPicker.css

```
@@ -0,0 +1,3 @@
+.rgb {
+    margin: 30px 0 50px 0;
```

Esto sería lo mismo:
```suggestion
    margin: 30px 0 50px;
```